apps/web-connect/src/features/checkout/components/PurchaseSuccessful.tsx

```diff
@@ -88,38 +88,59 @@ const PurchaseSuccessful: React.FC<IPurchaseSuccessful> = ({ returnToClient }) =
 							</div>
 						</a>
 					</div>
-					<div className="w-full text-elementalGrey text-[18px] flex justify-between border-t border-b border-chromaphobicBlack items-start px-[20px] py-[15px]">
-						<div className="flex flex-col items-end">
-							{address &&
-								<div className="relative">
-									<p className="text-base">Share your referral link with friends!</p>
-									<p className="text-base mb-3">They get 5% off and you get 15% of the sale.</p>
-									<div
-										className={`p-[1px] w-full h-full bg-chromaphobicBlack global-clip-btn`}>
-										<div
-											className={`relative items-center lg:w-[400px] w-full bg-optophobia h-fit flex justify-between text-americanSilver p-2 overflow-hidden text-lg font-medium global-clip-btn`}
-										>
-											<p className="select-none lg:max-w-[300px] sx:max-w-[210px] sm:max-w-[190px] overflow-x-hidden">{`${salePageBaseURL}/${address}`}</p>
-
-										</div>
-									</div>			
-									{canShare ? <div className="mr-3 cursor-pointer absolute right-[10px] sx:top-[73px] sm:top-[96px] z-60">
-										<ShareButton
-										buttonText={"Mint a key on the Xai to stake and start earning rewards and qualify for airdrops."}
-										buttonTitle={"5% Off Sentry Node Key Purchase"}
+					<div className="w-full text-elementalGrey text-[18px] flex flex-col border-t border-b border-chromaphobicBlack px-[20px] py-[15px]">
+						<p className="text-base">Share your referral link with friends!</p>
+						<p className="text-base mb-3">They get 5% off and you get 15% of the sale.</p>
+						<div className="p-[1px] w-full h-full bg-chromaphobicBlack global-clip-btn">
+						<div
+							className="relative flex items-center lg:w-[408px] w-full bg-optophobia h-fit p-2 gap-2 overflow-hidden text-lg font-medium global-clip-btn"
+						>
+							{/* Referral Link or Copied Text */}
+							{isTooltipAllowedToOpen ? (
+								<div className="text-americanSilver underline flex-grow">
+									Copied!
+								</div>
+							) : (
+								<a
+									href={`${salePageBaseURL}?promoCode=${address}`}
+									target="_blank"
+									rel="noopener noreferrer"
+									className="text-americanSilver underline hover:text-hornetSting truncate flex-grow"
+									onClick={(e) => {
+										e.preventDefault();
+										copyReferralCode();
+									}}
+								>
+									{`${salePageBaseURL}?promoCode=${address}`}
+								</a>
+							)}
+
+							{/* Share or Copy Icon */}
+							<div className="flex-shrink-0 flex items-center justify-center">
+								{canShare ? (
```

Comment:
Why share or Tooltip, this code is very hard to read.

I tested it and I did not see any tooltip, just the text conted changed to `Copied!` is that expected ?